doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -238,6 +238,10 @@ nRF Desktop
   * Application configurations for nRF54L05, nRF54L10, and nRF54L15 SoCs to use Fast Pair PSA cryptography (:kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_PSA`).
     Using PSA cryptography improves security and reduces memory footprint.
     Also increased the size of the Bluetooth receiving thread stack (:kconfig:option:`CONFIG_BT_RX_STACK_SIZE`) to prevent stack overflows.
+  * Application configurations for nRF52810 and nRF52820 SoCs to reduce memory footprint:
+
+    * Disabled Bluetooth long workqueue (:kconfig:option:`CONFIG_BT_LONG_WQ`).
```

Comment:
you can consider mentioning that now the system workqueue is responsible for long WQ tasks and that's why its stack is increased by this change

Reply:
I am not sure about it. IIRC, the stack sizes needed to generally be increased because of using PSA crypto APIs in Bluetooth (depending on application/sample various stacks needed to be increased). Maybe we could add a global note for whole NCS? (+ mention that stacks in apps/samples were aligned to new requirements)?